terraform/modules/daily_snapshot/service/newrelic_fail2ban.sh

```diff
@@ -22,13 +24,15 @@ include_matching_metrics:
     - regex "^syslog.*"
     - regex "^gpg-agent.*"
 metrics_network_sample_rate: -1
-metrics_process_sample_rate: 300
+metrics_process_sample_rate:-1
```

Comment:
I don't quite understand. This means we are disabling grabbing process metrics, right? Why? How much data does it generate? If it's really a lot, why do we still need `include_matching_metrics`?